js/majors.js

```diff
@@ -2213,19 +2162,19 @@ window.majors = {
           "18.104", "18.200", "18.204", "18.384", "18.424", "18.434", "18.504",
           "18.704", "18.784", "18.821", "18.904", "18.994"
         ],
-        [1,
+        [2,
           "18.104", "18.200", "18.204", "18.384", "18.424", "18.434", "18.504",
           "18.704", "18.784", "18.821", "18.904", "18.994",
-          "6.033", "8.06", "14.33", "18.100P", "18.100Q", "18.200", "18.642"
+          "8.06", "14.33", "18.100P", "18.100Q", "18.200", "18.642"
         ]
       ]
     ]
   },
 
-  m18_theoretical: {
-    name: "18 -- Mathematics (Pure Option) (as of Sep '17)",
+  m18_pure: {
```

Comment:
Renaming the keys will break CourseRoad for everyone who has roads saved with the old key. Can we leave it be for now?

Reply:
oops, yes, definitely. opened a PR to fix this